Assets/Tests/SpatialPersistence/Scripts/LoadScene.cs

```diff
@@ -0,0 +1,19 @@
+// Copyright (c) Ethar. All rights reserved.
```

Comment:
This script does not do anything special really. A one liner. I think this copyright is not needed here.